less.js

```diff
@@ -109,7 +109,11 @@ exports.translate = function(load) {
 				useFileCache = false;
 			}
 		})
-		.then(renderLess, renderLess);
+		.then(renderLess, renderLess)
+		.catch(function(err) {
+			console.error(err.message + " in " + err.filename + ":" + err.line);
+			console.dir(err);
```

Comment:
what is console.dir() doing here?

Reply:
console.dir displays a expandable list of all the properties of the `err`. This would allow users to dig deeper into the error if the filename and line were not enough for some reason.

Reply:
<img width="564" alt="screen shot 2017-11-20 at 10 29 57 am" src="https://user-images.githubusercontent.com/1085104/33029317-dc7b991a-cddd-11e7-807a-54129d2e4616.png">
Looks like this.

Reply:
👍